tasks/connectors/qualys_was/qualys_was.rb

```diff
@@ -81,7 +81,12 @@ def self.metadata
               type: "filename",
               required: false,
               default: "output/qualys_was",
-              description: "If set, will write a file upon completion. Path is relative to #{$basedir}" }
+              description: "If set, will write a file upon completion. Path is relative to #{$basedir}" },
+            { name: "match_ScannerIdentifier_with_vuln_def_name",
```

Comment:
Camel case is not used in toolkit parameters, for consistency I would suggest to use `match_scanner_identifier_with_vuln_def_name`

Alternatively, if we want to shorten it we could use something like `match_scan_id_with_vuln_def_name` or something a bit more general like `match_finding_with_vuln_def ` which speaks to the effect rather than the code changes.

Reply:
thanks! I will update the code according to the suggestion :)